elide-dbmanager/elide-dbmanager-multiplex/src/main/java/com/yahoo/elide/dbmanager/multiplex/MultiplexWriteTransaction.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2015, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.dbmanager.multiplex;
+
+import com.yahoo.elide.core.DatabaseManager;
+import com.yahoo.elide.core.DatabaseTransaction;
+import com.yahoo.elide.core.EntityDictionary;
+import com.yahoo.elide.core.exceptions.TransactionException;
+
+import com.google.common.collect.Lists;
+
+import java.io.IOException;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.IdentityHashMap;
+import java.util.List;
+import java.util.Map.Entry;
+
+import javax.ws.rs.core.MultivaluedHashMap;
+
+/**
+ * Multiplex transaction handler.
+ */
+public class MultiplexWriteTransaction extends MultiplexTransaction {
+    private static final Object NEWLY_CREATED_OBJECT = new Object();
+    private final IdentityHashMap<Object, Object> clonedObjects = new IdentityHashMap<>();
+    private final MultivaluedHashMap<DatabaseManager, Object> dirtyObjects = new MultivaluedHashMap<>();
+
+    public MultiplexWriteTransaction(MultiplexManager multiplexManager) {
+        super(multiplexManager);
+    }
+
+    @Override
+    protected DatabaseTransaction beginTransaction(DatabaseManager databaseManager) {
+        // begin updating transaction
+        return databaseManager.beginTransaction();
```

Comment:
Why do we need this method? Couldn't we just call `databaseManager.beginTransaction()` inline here: https://github.com/yahoo/elide/pull/26/files#diff-18fe3d40e18292c77178ad1194d82f0dR36?